aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -656,6 +703,11 @@ module aptos_framework::account {
     ) acquires OriginatingAddress {
         let address_map = &mut borrow_global_mut<OriginatingAddress>(@aptos_framework).address_map;
         let curr_auth_key = from_bcs::to_address(account_resource.authentication_key);
+        let new_auth_key = from_bcs::to_address(new_auth_key_vector);
+        assert!(
```

Comment:
Does this need to be an error? It's currently a no-op so this is effectively changing the semantics

Reply:
>  It's currently a no-op

Not necessarily, because it of the event emission which could result in unintutitive indexing results downstream